lib/engines/hcktest/tests.rb

```diff
@@ -309,7 +308,6 @@ def new_done
     def apply_filters
       @logger.info('Applying filters on finished tests')
       @tools.apply_project_filters(@tag)
-      sleep APPLYING_FILTERS_INTERVAL
```

Comment:
looks like this is not related to DHCPD. Please remove or move to separate commit if this is needed.

Reply:
I removed this. It is an irrelevant change.